### PR TITLE
`netstat -n` and `-r`

### PR DESCRIPTION
**Pull request by steelywing**

### CODE REVIEW

File: pages/linux/netstat.md [hunk 1]
```diff
@@ -21,3 +21,7 @@
 - List information continuously:
 
 `netstat -c`
+
+- List routes (-r) without resolve ip to hostname (-n):
```
Comment (waldyrious) [inline pages/linux/netstat.md]: "without resolving"

File: pages/linux/netstat.md [hunk 2]
```diff
@@ -21,3 +21,7 @@
...
+
+`netstat -nr`
```
Comment (waldyrious) [inline pages/linux/netstat.md]: How about matching the order that the options are mentioned in the description? I.e. "-rn".

**Comment by steelywing** [comment]
Sorry for my bad english :sweat_smile:

**Comment by rubenvereecken** [comment]
I usually don't like mentioning parameters explicit in the description, rather have it deductible from context. If you follow @waldyrious' last comment I think you can safely drop the `(-r)` and the other one.

**Comment by waldyrious** [comment]
Maybe if it was written "and do not reslolve ip..." even the -n could be removed :smile: 

By the way, IP should be capitalized.

**Comment by rubenvereecken** [comment]
That looks like intuitive enough, thanks for taking care of our comments, and thanks for a great addition to tldr pages!

**Comment by waldyrious** [comment]
:+1:

**Pull request merged**